qgispluginci/translation_clients/transifex.py

```diff
@@ -133,7 +133,9 @@ def get_translation(
             resource=self.get_resource(), language=language
         )
 
-        translated_content = requests.get(url).text
+        r = requests.get(url)
+        r.encoding = "utf-8"
+        translated_content = r.text
```

Comment:
Is the encoding not supposed to be defined by the response header (typically: `Content-Type: text; charset=utf-8`) which can be accessed using `r.encoding` in this case? If not enough, we can use `r.apparent_encoding` (https://docs.python-requests.org/en/latest/api/?highlight=apparent_encoding#requests.Response.apparent_encoding).

According to the documentation, the `r.text` property should already handle the encoding: https://docs.python-requests.org/en/latest/api/?highlight=apparent_encoding#requests.Response.text.

If any of these solutions, maybe the problem is coming from an incomplete or wrong response from remote server?

Forcing the encoding with a hard-coded value, even with utf-8, seems to be the last choice to me.

Reply:
On the current implementation the `Response.encoding` is `None` and the `apparent_encoding` is guessed (as well as the encoding of `text` when the `Response.encoding` is `None`) as `Windows-1254`. 

Honestly I'm not very familiar with this topic. So what do you suggest to solve it? When the response is wrong from remote server we might open an issue there (where?), but still we should have a solution here, since lots of translations have umlaut characters...

Reply:
considering the code is specific to Transifex and seems to fail, I don't see a big issue here to harcode this. Let's maybe just comment this in the code?

Reply:
Considered this in a comment.